src/CMakeLists.txt

```diff
@@ -2,18 +2,6 @@ if(MSYS)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DMSYS_PROCESS_USE_SH")
 endif()
 
-set(global_includes
-   ${Boost_INCLUDE_DIRS}
-   ${GTKMM_INCLUDE_DIRS}
-   ${GTKSVMM_INCLUDE_DIRS}
-   ${LIBCLANG_INCLUDE_DIRS}
-   ${ASPELL_INCLUDE_DIR}
-   ${LIBGIT2_INCLUDE_DIRS}
-   ../libclangmm/src
-   ../tiny-process-library
-   .
```

Comment:
Think it should be `../src` instead of `.` to make the tests compile. And juci will compile as well too then I guess.